crates/sncast/tests/helpers/fixtures.rs

```diff
@@ -69,6 +72,44 @@ pub async fn declare_contract(account: &str, path: &str, shortname: &str) -> Fie
     class_hash
 }
 
+pub async fn deploy_keystore_account() {
+    let keystore_path = "tests/data/keystore/deployed_key.json";
+    let account_path = "tests/data/keystore/deployed_account.json";
+    let private_key =
+        SigningKey::from_keystore(keystore_path, "123").expect("Could not get the private_key");
+
+    let provider = get_provider(URL).expect("Could not get the provider");
+    let chain_id = get_chain_id(&provider).await.expect("msg");
+
+    let contents = std::fs::read_to_string(account_path).expect("Failed to read accounts file");
```

Comment:
```suggestion
    let contents = std::fs::read_to_string(account_path).expect("Failed to read keystore account file");
```

keystore account file != accounts file